pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -248,6 +248,10 @@ private void startReceivingMessages(List<ConsumerImpl<T>> newConsumers) {
 
     private void receiveMessageFromConsumer(ConsumerImpl<T> consumer) {
         consumer.receiveAsync().thenAcceptAsync(message -> {
+            if (consumer.isDuringSeek()) {
+                receiveMessageFromConsumer(consumer);
```

Comment:
This will introduce a busy loop here?

Reply:
Maybe not. But after adding the following code change to simulate a large latency in `clearReceiverQueue`:

```java
    private BatchMessageIdImpl clearReceiverQueue() {
        /* ... */

        try {
            Thread.sleep(500);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
```

The test failed again.

Reply:
Increase timeout to 4 in test.